TrkPatRec/fcl/prolog.fcl

```diff
@@ -27,8 +27,8 @@ TimeClusterFinder : {
   CaloClusterCollection : "CaloClusterFromProtoCluster"
   # enable calorimeter cluster info in time clustering
   CaloClusterCollection : "CaloClusterFromProtoCluster"
-  UseCaloCluster : true
-  UseCaloClusterPosition : true
+  UseCaloCluster : false
```

Comment:
Why is this changed?

Reply:
I made some tests but it's back to true in my code.... Not sure why that happened. I'll re-commit and see if it changes back to true.